Config/Routes.js

```diff
@@ -4,6 +4,7 @@ function routes(app) {
     app.get('/products', ProductController.getAllProducts)
     app.post('/products', ProductController.addSingleProduct)
     app.get('/products/:SKU', ProductController.getSingleProduct)
+    app.put('/products/', ProductController.updateStockLevel)
```

Comment:
Should this be '/products/:SKU'? Currently the SKU is being read from the request body, but in the documentation it says going to be defined in the URL

Reply:
How do you define PUT in the url? it requires stock_level as well as SKU for the request